dts/bindings/sensor/sbs-gauge.yaml

```diff
@@ -0,0 +1,5 @@
+description: Smart Battery Fuel Gauge
+
+compatible: "sbs-gauge"
```

Comment:
Can you also beef up the description a bit? This will show up in the bindings index in the DT reference documentation. I'm assuming this is related to http://www.sbs-forum.org/specs/; it would be good to provide the user with a bit more context either way.

@galak any thoughts on the compatible here?

Reply:
added the related link to the description